parser/src/powdr.lalrpop

```diff
@@ -233,6 +234,13 @@ Instruction: ASMStatement<T> = {
     <@L> <Identifier> <ExpressionList> ";" => ASMStatement::Instruction(<>)
 }
 
+DebugDirective: ASMStatement<T> = {
+    <l:@L> "debug" "file" <n:Integer> <d:StringLiteral> <f:StringLiteral> ";"
```

Comment:
Do we want to have commas as separators or not? We usually have them because there can be complex expressions, but here, we need just direct numbers and strings. For the same reason, there are no commas in gcc assembly either.